### ISSUE TITLE
Not allow viewers moving forward before having input to much effort on the edition part

### ISSUE THREAD

**Issue by LucieGueuning** [MSFREACH/msf-reach#432]

**Comment by matthewberryman** [comment]
Done, we removed the restrictions on event input

**Comment by matthewberryman** [comment]
1. On event creation / event edit, make 1 API call (need dummy endpoint) to check group ID in operators and then
2.  flag to to the user via message (top of modal somewhere) that they need to contact @LucieGueuning if they would like operator access.

**Comment by matthewberryman** [comment]
(If we could have breakdown of API changes and initial display by @mehrdadgit and then any ui tidy up by @qclin I think that's a sensible breakdown)

**Comment by qclin** [comment]
@matthewberryman i think what Lucie refers to is this moment during the user Interview when Mick had enter data in one section but then he scroll further down and forgot which section he had 'dirtied' and keeps getting the alert popup. I think this is a design problem. we can simply scroll to the section he needs to save first

**Comment by matthewberryman** [comment]
@qclin no I spoke with Lucie on the call last night and this is about read-only users entering information, but not being able to see until the submission that they don't have access and need to write to Lucie to request access.

**Comment by LucieGueuning** [comment]
@matthewberryman and @qclin, for clarification, this is well two separate issues.

**Comment by LucieGueuning** [comment]
Not working. Viewers can create event or edit anything on the platform at the moment. There is no restrictions for them anymore. This is a real issue.

**Comment by matthewberryman** [comment]
On test or prod? And, if prod, is it actually letting a viewer create an event or just not flagging the warning (what this actual issue is about)

There are some restrictions I removed on test account on test environment only due to limitations coming from the AD side out of my control.

Are the messages about no operator access showing up for you?

Sent from my iPhone

On 14 Nov 2018, at 05:03, Lucie Gueuning <notifications@github.com<mailto:notifications@github.com>> wrote:


Not working. Viewers can create event or edit anything on the platform at the moment. There is no restrictions for them anymore. This is a real issue.

**Comment by LucieGueuning** [comment]
*   This is on test as I don’t want to test to create event from viewer account and that could work on prod.
  *   In test, it is actually letting the viewer create an event.
  *   The message is not showing up for me as I am operator. The problem is that for viewer, in test site, the message for no operator access, is not showing up neither.

**Comment by matthewberryman** [comment]
Yes not showing up for test account on test, because I needed to make that change on test, test has write access for test user on test environment only, so the message doesn’t show up there, so please test using test account on prod environment.

Sent from my iPhone

On 14 Nov 2018, at 06:04, Lucie Gueuning <notifications@github.com<mailto:notifications@github.com>> wrote:

* This is on test as I don’t want to test to create event from viewer account and that could work on prod.
* In test, it is actually letting the viewer create an event.
* The message is not showing up for me as I am operator. The problem is that for viewer, in test site, the message for no operator access, is not showing up neither.

**Comment by matthewberryman** [comment]
@LucieGueuning, to clarify:
* The test account is an operator on test environment (so I can test things).
* To test the message using the test account you must use the test account on the prod environment, where test is not an operator, only a viewer. I have confirmed—by double checking the code and configuration that remain unchanged for that aspect, and testing on prod, being ready to delete the test event just in case—that even if I click all the way through event creation it remains the case that the test account cannot create events on prod.

You should see the following message as test user on prod environment, which is the only change that I've made, per this specific issue:
<img width="1160" alt="screenshot 2018-11-14 12 37 54" src="https://user-images.githubusercontent.com/2288238/48454117-2c93c980-e80a-11e8-93ea-ba9fe7422575.png">

**Comment by matthewberryman** [comment]
Update: to test #409 deletion of public contacts by operators, I have temporarily removed test account as an operator on test environment, but your usual MSF account should still be an operator on test environment.  That will also help testing on this issue as you can (until further notice) use test account on test to check this one.

**Comment by LucieGueuning** [comment]
feedback: 
- great solution :)
- second red sentence to update: To get operator permission, please contact us (hyperlink to my email address). This is to keep consistency with the sentence once you want to create an event and the system doesn't allow you. 
- also having that red sentences for any features for edit access. For example, the edition of an event, confirming/unconfirming/ignoring reports, deleting event, deleting public contact, etc. Viewers even should not have the red cross to delete content of REACH. 
![image](https://user-images.githubusercontent.com/25951555/48488709-4a1c5f80-e819-11e8-9781-1f351a48270c.png)
- any other features that viewers should require operator access, should have this red sentences at the beginning of the work to avoid frustration.
